assets/src/blocks/Happypoint/HappypointBlock.js

```diff
@@ -8,6 +8,7 @@ export class HappypointBlock {
 
     registerBlockType('planet4-blocks/happypoint', {
       title: __('Happypoint', 'planet4-blocks-backend'),
```

Comment:
This however should not be translated, if I'm not mistaken we decided to keep the block names untranslated to make it easier for documentation, but it makes sense to me that the block descriptions should be translatable!

Reply:
Fixed!